spec/rails_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'simplecov'
```

Comment:
oh yes thanks for adding this!

Reply:
Yupp, surprised we didn't see this til now! Good add